### ISSUE TITLE
Site Address

### ISSUE THREAD

**Issue by Beianrui** [sih4sing5hong5/google-sites-liberation#130]
When trying to export from sites.google.com it keeps adding /feeds/content/site to the URL string and no matter what I change in the fields it adds this. 
Any Suggestions on what I might do to stop it from doing this and get it to go directly to the site I wan
No data returned. Can you get anything from https://sites.google.com/a/[site].com/feeds/content/site/tsknowledge/.

**Comment by ethanhuang03** [comment]
I also have this problem

**Comment by vinmanp** [comment]
Were you able to overcome this problem?

**Comment by ethanhuang03** [comment]
> Were you able to overcome this problem?

No, but do you think you can change the code so it would be something like “https://sites.google.com/view/[insert site name]/“ should work. I think.

**Comment by grahamjohnson10** [comment]
I have the same problem, has this ever been resolved.
